key-manager/src/main/scala/org/bitcoins/keymanager/bip39/BIP39KeyManager.scala

```diff
@@ -37,7 +37,8 @@ case class BIP39KeyManager(
   private val privVersion: ExtKeyPrivVersion =
     HDUtil.getXprivVersion(kmParams.purpose, kmParams.network)
 
-  private val rootExtPrivKey = seed.toExtPrivateKey(privVersion)
+  // FIXME Temporary public, should be private
+  val rootExtPrivKey: ExtPrivateKey = seed.toExtPrivateKey(privVersion)
```

Comment:
outdated comment?

Reply:
This is still required to make the `BinaryOutcomeDLCClient` from the wallet, can change back to `private val` when we change the client to use a `Sign` instead

Reply:
could you leave the detail of what needs to change for this fixme in the comment?